src/sensai/torch/torch_data.py

```diff
@@ -238,10 +238,26 @@ def _inputOutputPairs(self, indices):
 class TorchDataSet:
     @abstractmethod
     def iterBatches(self, batchSize: int, shuffle: bool = False, inputOnly=False) -> Generator[Union[Tuple[torch.Tensor, torch.Tensor], torch.Tensor], None, None]:
```

Comment:
Since I will probably write some data loaders for images anyway, I wonder whether this can be geneIralized a bit and used in the rest of sensai. For example, we might have a base DataSet that generates batches of numpy arrays and convert them to tensors either in a child class or directly in the methods consuming the dataset

Reply:
I you agree, we can follow up in a separate issue

Reply:
Yes, we can add such an issue, it would make sense. But how about batches of pandas.DataFrames?